### ISSUE TITLE
Reaction position and color is wrong and confusing sometimes

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/talk-android#4557]
see 4 messages:
- Own text
- Own file
- Foreign text
- Foreign file

You (Me) like them all | Someone (Nickv2) else likes them all
---|---
<img src="https://github.com/user-attachments/assets/f5c4b5f9-8d8d-42d4-b050-78401cbd4133" height="400"> | !<img src="https://github.com/user-attachments/assets/b3392795-e4c2-4f99-850a-2980e0338908" height="400">

## Expected
- Consistent feeling that you liked something (primary color)

## Actual
- File reactions are "outside of the bubble"
- Own reactions on foreign files have no color background and look exactly like reactions from some one else
- Own reactions on own messages have a different color then on foreign messages

**Comment by nickvergessen** [comment]
@sowjanyakch please have a look

**Comment by AndyScherzinger** [comment]
So originally we themed the reaction background for own/others' messages differently, so that should be easy to fix by always theming it with primary color.

File reactions are outside of "the bubble because file-messages do not have a bubble at all at the moment at least on Android. Yet that could come later and wouldn't influence the reaction backgrounds regarding the use of primary color for won reactions.

**Comment by nimishavijay** [comment]
True, all messages liked by you should have the primary color like the reaction to NickV2's "Other text" message in `You (me) like them all` 

The position should also stay consistent for all, the reactions for the files are too much sticking to the edge. It should be the same as the reactions for just text. 

The position of the reactions for just text can also be adjusted to be a couple pixels farther from the text in the bubble. 
The top+bottom padding can be reduced by 1 or 2px and the right+left padding can be increased by 1or 2px to make it look less round.

**Comment by AndyScherzinger** [comment]
I _very_ strongly advice against the primary color decision, because this logic with a different color on own messages is in place for a reason, see the change in action on sermo:

![Image](https://github.com/user-attachments/assets/d1a6a42a-0518-42b4-bb1f-b77b51c8d908)

-> this doesn't work at all, contrast-wise (in any case)
